tests/valgrind-check/run.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+set -e
+trap "echo FAILURE" ERR
+
+set -x
+
+cd ../../../
+
+if which podman ; then
+  CLI="podman --cgroup-manager=cgroupfs"
+  [ "`whoami`" == "root" ] || { echo "Error: Must be root (sudo) with podman" ; exit 1 ; }
```

Comment:
Please use good old `=` for "equals", this is not C. :)

Reply:
This is the first result when googling "String equals bash":

https://linuxhint.com/compare_strings_bash/#:~:text=You%20can%20check%20the%20equality,two%20strings%20also%20in%20bash.